# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/fields/mixed_methods.js
# Commit: 1b8c42577e24
# Full Hash: 1b8c42577e2410dbbe050bb30b0ccced9d7c3771
# Author: Ashley Hauck <khyperia@mozilla.com>
# Date: 2019-03-20 10:52:37
# Regressor Bug: 1535166
# File Overlap Count: 1
# Description:
#   Bug 1535166 - Implement computed field names. r=jorendorff
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D23408
# ==============================================================================

diff -r 28370bf14b97 -r 1b8c42577e24 js/src/jit-test/tests/fields/mixed_methods.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/src/jit-test/tests/fields/mixed_methods.js	Tue Mar 19 20:23:03 2019 +0000
@@ -0,0 +1,11 @@
+// |jit-test| --enable-experimental-fields
+
+class C {
+    x;
+    y(){}
+    z = 2;
+    w(){};
+}
+
+if (typeof reportCompare === "function")
+  reportCompare(true, true);